agentconfig.yaml

```diff
@@ -0,0 +1,13 @@
+schedule: "* * * * *"
```

Comment:
We already have agent.yaml - might be worth removing that file?

Reply:
I added that file by mistake. It should be deleted now.